.github/workflows/ci.yml

```
@@ -20,7 +20,7 @@ jobs:
           cue-version: 0.11.0
```

```suggestion
          version: 0.11.0
```

Gonna give it a run with this change, per CI warning